### ISSUE TITLE
Profile format?

### ISSUE THREAD

**Issue by jvgaultney** [silnrsi/ufochecks#5]
What format should we use for the profiles? YAML? JSON? Python data structure? Please express your preference!

**Comment by madig** [comment]
If the format is not supposed to house code (e.g. for actions), YAML or https://github.com/crdoconnor/strictyaml is probably an okay choice.

**Comment by graphicore** [comment]
I usually prefer YAML over JSON, because YAML has comments, this could be handy to clarify profile details. It also is a superset of JSON.

**Comment by jvgaultney** [comment]
Yes, comments would be very useful. YAML is also about as human-parsable as you can get.
